### PR TITLE
[fix](fe) add fe isReady check before getMasterIp

### PR DESCRIPTION
**Pull request by nextdreamblue**
# Proposed changes

Issue Number: close #xxx

## Problem summary

when fe node is not ready, will get "" for master ip, and redirect will get error

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by nextdreamblue** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.99 seconds
 stream load tsv:          428 seconds loaded 74807831229 Bytes, about 166 MB/s
 stream load json:         23 seconds loaded 2358488459 Bytes, about 97 MB/s
 stream load orc:          73 seconds loaded 1101869774 Bytes, about 14 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230406060927_clickbench_pr_125327.html

File: fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/GetLoadInfoAction.java [hunk 1]
```diff
@@ -77,9 +77,13 @@ public Object execute(
             return new RestBaseResult("No cluster selected");
         }
 
-        RedirectView redirectView = redirectToMaster(request, response);
-        if (redirectView != null) {
-            return redirectView;
+        try {
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/GetLoadInfoAction.java:80]: I think we'd better extract a common method to handle this.

**Comment by nextdreamblue** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**